### PR TITLE
Text call to action component

### PR DESCRIPTION
**Pull request by EphraimAssi**
# Description

[Create text with call to action component](https://trello.com/c/sAiv36Ew/187-187-create-text-with-call-to-action-component)

Create a text with call to action component so that we can dynamically reference them according to the data in the CMS.
image

## Acceptance Criteria

Should be able to pass in CSS classes in order to change sizing i.e. full width half width etc

## Test Instructions

1. Test the page in different screen sizes
2. Compare the component with the Figma mock-up
3. Check the code to see if there's anything to improve
4. Check if add different widths size in the className Prop work.

## Checklist

- [x] Strings use placeholders for translation (No hard coded strings)
- [x] Unit tests have been added/updated

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: components/molecules/TextButtonField.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          id={props.idButton}
+          className={"mt-2 text-xs md:text-base"}
+          text={props.buttonText}
+          secondary={props.secondary}
+          disabled={props.disabled}
+          custom={props.custom}
+          href={props.href}
+          dataCyButton={props.dataCyButton}
+        />
+      ) : null}
```
Comment (Moro-Code) [inline components/molecules/TextButtonField.js:34]: do not pass null. Should either be undefined or an empty string

File: components/molecules/TextButtonField.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+/**
+ *  component
+ */
+export function TextButtonField(props) {
+  return (
+    <div className={props.className + " mb-4"}>
+      <h1>{props.title}</h1>
+
+      {props.text.map((paragraph, key) => {
```
Comment (Moro-Code) [inline components/molecules/TextButtonField.js:12]: Follow the example of setting markdown generated HTML from the NextJS tutorial. Array of text will not be passed in. Just HTML that needs to be set

File: components/molecules/TextButtonField.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            key={key}
+            className="my-7 text-sm md:text-p leading-normal text-left font-normal font-body"
+          >
+            {paragraph}
+          </p>
+        );
+      })}
+
+      {props.buttonText ? (
+        <ActionButton
```
Comment (Moro-Code) [inline components/molecules/TextButtonField.js:24]: Let's make the secondary styling the default here

File: components/molecules/TextButtonField.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+
+/**
+ *  component
+ */
+export function TextButtonField(props) {
+  return (
+    <div className={props.className + " mb-4"}>
```
Comment (Moro-Code) [inline components/molecules/TextButtonField.js:9]: Hmmm not sure this is working. 
Screen Shot 2021-05-11 at 1 07 11 PM

File: components/molecules/TextButtonField.test.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+import {
+  Primary,
+  Secondary,
+  Disabled,
+  Custom,
+} from "./TextButtonField.stories";
+
+expect.extend(toHaveNoViolations);
+
+describe("TextButtonField", () => {
```
Comment (Moro-Code) [inline components/molecules/TextButtonField.test.js:13]: You need to flesh out your testing here. Is the button rendered in the correct styles ? Are you seeing the classes you pass in ? Is all the text rendered correctly on the page ?

**Review by Moro-Code — CHANGES_REQUESTED** [review]
Good work overall !! There are some changes needed, let me know if you have any questions

**Comment by EphraimAssi** [comment]
Changing the width seems to be working on my computer

**Review by Moro-Code — APPROVED** [review]
Great work @EphraimAssi !!

**Pull request merged**